cmd/rook/object/connection.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package object
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/rook/rook/cmd/rook/rook"
+	"github.com/rook/rook/pkg/rook/client"
+	"github.com/rook/rook/pkg/util/flags"
+	"github.com/spf13/cobra"
+)
+
+var connectionCmd = &cobra.Command{
+	Use:     "connection",
+	Short:   "Gets connection information that will allow a client to access object storage in the cluster",
+	Aliases: []string{"conn"},
+}
+
+func init() {
+	connectionCmd.RunE = getConnectionInfoEntry
+}
+
+func getConnectionInfoEntry(cmd *cobra.Command, args []string) error {
+	rook.SetupLogging()
+
+	if err := flags.VerifyRequiredFlags(cmd, []string{}); err != nil {
+		return err
+	}
+
+	c := client.NewRookNetworkRestClient(client.GetRestURL(rook.APIServerEndpoint), http.DefaultClient)
+	out, err := getConnectionInfo(c)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	fmt.Print(out)
+	return nil
+}
+
+func getConnectionInfo(c client.RookRestClient) (string, error) {
+	connInfo, err := c.GetObjectStoreConnectionInfo()
+
+	if err != nil {
+		if client.IsHttpNotFound(err) {
+			return "object store connection info is not ready, if \"object create\" has already been run, please be patient\n", nil
```

Comment:
this is just a temporary message anyways until we get #251, which I think will be soon.